docs/workflows/transitions/preprocess/preprocess/utils.py

```diff
@@ -114,3 +121,77 @@ def merge_predictions_and_gt(predictions, old_ground_truth, field_config):
 
     return updated_predictions
 
+
+def overlap(last_lines, first_lines):
```

Comment:
It returns true if there is an overlap between the labels. If some of the values for the same field are different (for instance line_1[description] = 'abc' and line_2[description] = 'cdf') we return false.

It will return true of the value of the fields are the same across the fields (including if a label is missing for both lines), or if a label is missing from one line, but present in the other.

I will add a comment.